src/Button/Button.js

```diff
@@ -25,6 +25,7 @@ export const styleSheet = createStyleSheet('Button', (theme) => {
       color: palette.text.primary,
       backgroundColor: 'transparent',
       transition: transitions.multi(['background-color', 'box-shadow'], '250ms'),
+      willChange: 'transform',
```

Comment:
I don't think that we should follow that path. The `willChange` property should be used when the property is likely to change in the near future. I don't think it's relevant here.
I appreciate your effort but I think that our resources would be better spent on other issues.